### PR TITLE
Convert release_scraper tests to pytest

### PR DESCRIPTION
**Pull request by Utkarsh1308**

### CODE REVIEW

**Comment by Utkarsh1308** [comment]
@whimboo please review

File: tests/release_scraper/test_release_scraper.py [hunk 1]
```diff
@@ -7,127 +7,64 @@
...
-     'filename': 'firefox-23.0.1.en-US.mac.dmg',
-     'url': 'firefox/releases/23.0.1/mac/en-US/Firefox 23.0.1.dmg'},
-    # -a firefox -p win32 -v 23.0.1 -l de
-    {'args': {'application': 'firefox',
-              'locale': 'de',
-              'platform': 'win32',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.de.win32.exe',
-     'url': 'firefox/releases/23.0.1/win32/de/Firefox Setup 23.0.1.exe'},
-    # -a firefox -p win32 -v 23.0.1 --stub (old format)
```
Comment (whimboo) [inline tests/release_scraper/test_release_scraper.py]: Similar to the other tests you worked on for the candidate scraper, please keep the comments about the old and new stub installer format.

File: tests/release_scraper/test_release_scraper.py [hunk 2]
```diff
@@ -7,127 +7,64 @@
...
+     'thunderbird-17.0.en-US.mac.dmg',
+     'thunderbird/releases/17.0/mac/en-US/Thunderbird 17.0.dmg'),
+    ({'application': 'thunderbird', 'locale': 'de', 'platform': 'win32', 'version': '17.0'},
+     'thunderbird-17.0.de.win32.exe',
+     'thunderbird/releases/17.0/win32/de/Thunderbird Setup 17.0.exe'),
+])
 
-    def test_scraper(self):
-        """Testing various download scenarios for ReleaseScraper"""
+def test_scraper(httpd, tmpdir, args, filename, url):
```
Comment (whimboo) [inline tests/release_scraper/test_release_scraper.py]: Lets rename the test to `test_release_scraper`.

**Comment by whimboo** [comment]
The changes look fine now but Appveyor failed to run some tests.  I will retrigger the job, and if it passes we can merge your PR. Thanks a lot!

**Pull request merged**